src/js/discharge-wizard/components/FormQuestions.jsx

```diff
@@ -16,6 +16,8 @@ const scroller = Scroll.scroller;
 class FormQuestions extends React.Component {
   componentDidMount() {
     Scroll.animateScroll.scrollToTop();
+    const focusElement = document.getElementById('dw-home-link');
+    if (focusElement) { focusElement.focus(); }
   }
 
```

Comment:
There's a helper for focusing elements that was suggested during code review for the user feedback PR. Haven't looked into if there are any advantages but it's here -

```jsx
import { focusElement } from '../../common/utils/helpers';

...
focusElement(selector);
```